_blog-src/_posts/2015-06-15-apiman-msa-day.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Microservices Architecture Day Appearance"
+date:   2015-06-15 17:30:00 +0100
+author: marc_savy
+categories: microservices conference talk
+---
+
+I had the pleasure of presenting on apiman at the recent [Microservices Architecture Developer Day], with our Fabric8 friend Kurt delivering a short demo of apiman in Fabric8. It was particularly enjoyable meeting developers who are interested in, or are already using, apiman - so, thank you for your insightful questions both during, and after, the presentation.
+
+Given the packed schedule, there was a limited amount of time to explore apiman plus microservices, and hence this seems like a good opportunity to write a blog post expanding upon the themes I touched upon.
+
+So, if you're interested in understanding the value the API management can have in a microservices architectures; please, read on!
+
+////
+Use this pattern to avoid asciidoctor rendering the HTML comment.
+<!--more-->
+////
+
+== Services, services, everywhere!
+
+A microservices architecture encourages the decomposition of problems into small, isolated and reusable chunks; much in the same spirit as UNIX's _do one job and do it well_ philosophy. This brings with it a whole raft of benefits, for instance: we can scale each component independently, avoiding unnecessarily dragging along other components which just happen to reside in the same monolith; we can easily have a single team wholly responsible for a single microservice's lifecycle, simplifying debugging and allowing them to use their skills and technologies of choice; and, we can easily create new functionality via composition. However, this proliferation of heterogeneous services poses some challenges, a significant subset of which API Management is a key solution to.
+
+== Avoiding reimplementation of cross-cutting functionality
+
+A problem that can quickly become apparent is that there are often key pieces of important functionality that is difficult or undesirable to reimplement in every API. Good examples of this would be security concerns like authentication and authorization, or business concerns like rate-limiting, billing and auditing.
```

Comment:
Have reworded this to 

A problem that can quickly become apparent is that there are key pieces of important common functionality that are difficult or undesirable to reimplement in every API. ...